lib/node_modules/@stdlib/math/base/special/sqrtpif/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env julia
+#
+# @license Apache-2.0
+#
+# Copyright (c) 2024 The Stdlib Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import JSON
+
+"""
+    gen( x, name )
+
+Generate fixture data and write to file.
+
+# Arguments
+
+* `x`: domain
+* `name::AbstractString`: output filename
+
+# Examples
+
+``` julia
+julia> x = range( -1000, stop = 1000, length = 2001 );
+julia> gen( x, \"data.json\" );
+```
+"""
+function gen( x, name )
+	y = Array{Float32}( undef, length(x) );
+	for i in eachindex(x)
+		y[i] = sqrt( Float32( x[i] * pi ) );
```

Comment:
@gururaj1512 I tried running these tests locally. In order to fix your failing tests, you'll need to change this to:
```suggestion
		y[i] = sqrt( Float32( Float32( x[i] ) * pi ) );
```
We first cast `x[ i ]` to `Float32`, and then cast its product with `pi` to `Float32` again.
After changing this, you'll need to regenerate the fixtures, and all tests should pass after this.

Reply:
Thanks for clarifying. This is working well.